hadoop-ozone/dist/src/shell/ozone/ozone-functions.sh

```diff
@@ -1433,7 +1433,7 @@ function ozone_set_module_access_args
     OZONE_MODULE_ACCESS_ARGS="${OZONE_MODULE_ACCESS_ARGS} --add-exports java.management/com.sun.jmx.mbeanserver=ALL-UNNAMED"
   fi
   if [[ "${JAVA_MAJOR_VERSION}" -ge 9 ]]; then
-    OZONE_MODULE_ACCESS_ARGS="${OZONE_MODULE_ACCESS_ARGS} --add-opens java.base/java.nio=ALL-UNNAMED"
+    OZONE_MODULE_ACCESS_ARGS="${OZONE_MODULE_ACCESS_ARGS} --add-opens='java.base/java.nio=ALL-UNNAMED' --add-opens='java.base/java.lang=ALL-UNNAMED' --add-opens='java.base/java.lang.reflect=ALL-UNNAMED' --add-opens='java.management/com.sun.jmx.mbeanserver=ALL-UNNAMED' --add-exports='java.management/com.sun.jmx.mbeanserver=ALL-UNNAMED' --add-exports='java.base/sun.net.dns=ALL-UNNAMED' "
```

Comment:
1. Please follow the existing pattern to append items to the variable, one per line.  Do not use `'`.
2. This `-ge 9` block also applies to Java 17+, so we should not have duplicates in the two `if` blocks.  If something is needed for 9+ is already there in the `-ge 17` block, move it.
3. Is `java.base/sun.net.dns` required for fixing reflection?